modules/system/console/asset/AssetCreate.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+namespace System\Console\Asset;
+
+use Closure;
+use Cms\Classes\Theme;
+use Symfony\Component\Console\Input\InputOption;
+use System\Classes\Asset\BundleManager;
+use System\Classes\Asset\PackageJson;
+use System\Classes\Asset\PackageManager;
+use System\Classes\PluginManager;
+use Winter\Storm\Console\Command;
+use Winter\Storm\Support\Facades\File;
+
+abstract class AssetCreate extends Command
+{
+    protected const TYPE_THEME = 'theme';
+    protected const TYPE_PLUGIN = 'plugin';
+
+    /**
+     * @var string The console command description.
+     */
+    protected $description = 'Creates the compiler configuration files for the provided package and optionally installs the necessary dependencies for any selected asset bundles.';
+
+    /**
+     * Local cache of fixture path
+     */
+    private string $fixturePath;
+
+    /**
+     * The type of compilable to configure
+     */
+    protected string $assetType;
+
+    /**
+     * The name of the config file
+     */
+    protected string $configFile;
+
+    /**
+     * Dynamically generate options for all available bundles
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        foreach (BundleManager::instance()->getBundles() as $bundle) {
+            $this->addOption($bundle, null, InputOption::VALUE_NONE, 'Create ' . $bundle . ' configuration');
+        }
+    }
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): int
+    {
+        $package = $this->argument('packageName');
+
+        $this->fixturePath = __DIR__ . '/fixtures/config';
+
+        $compilableAssets = PackageManager::instance();
+        $compilableAssets->fireCallbacks();
+
+        $packages = $compilableAssets->getPackages($this->assetType, true);
+
+        if (
+            isset($packages[$package])
+            && !$this->confirm('Package `' . $package . '` has already been configured, are you sure you wish to continue?')
+        ) {
+            return 1;
```

Comment:
A user cancelling the action should not be an exit 1. It's an intended action, so it should be exit 0.
```suggestion
            return 0;
```